magellan-legacy/src/main/java/com/wealthfront/magellan/Navigator.kt

```diff
@@ -54,26 +55,35 @@ public class Navigator internal constructor(
     delegate.navigate(direction, backStackOperation)
   }
 
-  @JvmOverloads
-  public fun replace(navigable: NavigableCompat, magellanTransition: MagellanTransition? = null) {
```

Comment:
Do default parameters cause problems for mocking? I think I remember something about that

Reply:
yeah. that was it. So I decided to break them apart into separate cascading methods instead of using `JvmOverloads` which doesn't seem to be working well with mocking